### ISSUE TITLE
[serendipity_event_markdown] Add #ids to headings

### ISSUE THREAD

**Issue by th-h** [s9y/Serendipity#729]
Auto-create HTML ids (#somename) for all `<h?>` elements so it's possible to link to them from an auto-generated TOC. See https://board.s9y.org/viewtopic.php?f=3&t=24825

**Comment by th-h** [comment]
Should be fixed by https://github.com/s9y/additional_plugins/commit/2ce165815493bdc75eb91ef9b9b8d2756d35fa7c

**Comment by aisbergde** [comment]
will this be promoted via spartacus? Or how could I test this?

**Comment by aisbergde** [comment]
wurde hier schon beantwortet:
https://board.s9y.org/index.php?mark_notification=2304&hash=d77ddec4

**Comment by th-h** [comment]
> will this be promoted via spartacus? Or how could I test this?

Yes. Spartacus does an update run every day between midnight and 01:00 AM; after that you can update your plugin from Serendipity. (Or you can download https://raw.githubusercontent.com/s9y/additional_plugins/master/serendipity_event_markdown/serendipity_event_markdown.php and replace your version of `serendipity_event_markdown.php` in `.../plugins/serendipity_event_markdown` with that file.

**Comment by aisbergde** [comment]
There is still a small problem:
I use the VS Code plugin "Markdown All in One" to create the TOC. 

When I create the TOC the "/" is removed from the label. But when the markdown is converted into html the "/" is replaced by "-". And now they do not match.

Maybe you could also remove the "/" and not replace it?

generated Markdown:
```
  * [02/2020 - bis jetzt; aaa](#022020---bis-jetzt-aaa)
```

```
<li><a href="#022020---bis-jetzt-aaa">02/2020 - bis jetzt; aaa</a></li>

<h2 id="02-2020-bis-jetzt-aaa">02/2020 - bis jetzt; aaa</h2>

```

**Comment by aisbergde** [comment]
Und Umlaute sollten auch erhalten bleiben, here in the word "für":

```
* [Warum ich den AnalyticsCreator nicht für die Erstellung multidimensionaler Modelle verwende](#warum-ich-den-analyticscreator-nicht-für-die-erstellung-multidimensionaler-modelle-verwende)
```

```
<li><a href="#warum-ich-den-analyticscreator-nicht-für-die-erstellung-multidimensionaler-modelle-verwende">Warum ich den AnalyticsCreator nicht für die Erstellung multidimensionaler Modelle verwende</a></li>
<h2 id="warum-ich-den-analyticscreator-nicht-f-r-die-erstellung-multidimensionaler-modelle-verwende">Warum ich den AnalyticsCreator nicht für die Erstellung multidimensionaler Modelle verwende</h2>
```

**Comment by aisbergde** [comment]
Klammern sollten hingegen entfernt werden:

```
    * [Definition von SSAS-Tabellen-Attributen (Spalten)](#definition-von-ssas-tabellen-attributen-spalten)
```

**Comment by th-h** [comment]
Okay, I see. Will have a look again these days.
